target/linux/brcmiproc/image/Makefile

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright (C) 2016-2017 Yousong Zhou <yszhou4tech@gmail.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+include $(TOPDIR)/rules.mk
+include $(INCLUDE_DIR)/image.mk
+
+define Device/Default
+  DEVICE_DTS_DIR := $(DTS_DIR)/broadcom
+endef
+
+define Device/bcm958802a802x
```

Comment:
Devices should be defined by vendor_model scheme.

Reply:
Used the board's vendor name